src/event_timer.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2021 Canonical
+# See LICENSE file for licensing details.
+
+"""EventTimer for scheduling dispatch of juju event on regular intervals."""
+
+import subprocess  # nosec B404
```

Comment:
Would it be worth including a comment here why subprocess is needed?

Reply:
I was think the reasoning should be documented at each location where `subprocess` is used.
If no `subprocess` is used the import will cause linting for unused import.

There is some advantage to this.
- Line comments cause formatter to add an extra black line between imports.
- Each instance of `subprocess` usage should have it own reason. Since usage throughout a file may be different.